ihatemoney/tests/tests.py

```diff
@@ -281,6 +281,46 @@ def test_project_creation(self):
             # no new project added
             self.assertEqual(len(models.Project.query.all()), 1)
 
+    def test_project_creation_without_public_permissions(self):
+        self.app.config["ALLOW_PUBLIC_PROJECT_CREATION"] = False
+        with self.app.test_client() as c:
+            # add a valid project
+            c.post(
+                "/create",
+                data={
+                    "name": "The fabulous raclette party",
+                    "id": "raclette",
+                    "password": "party",
+                    "contact_email": "raclette@notmyidea.org",
+                },
+            )
+
+            # session is not updated
+            self.assertNotIn("raclette", session)
+
+            # project is created
```

Comment:
I believe the comment should state « not created ».

Reply:
uops, you are right!, I change it and upload the commit.

Reply:
ready